File: Player.cpp
//Honor Pledge:
//
//I pledge that I have neither given nor
//received any help on this assignment.
//
//jascho
#include "Player.h"
#include <stdlib.h>
#include <iostream>
Player::Player()
{
	Horse= new char[5];
	this->shot = 0;
	this->score = false;
}
Player::~Player()
{
	delete [] Horse;
}
int Player::setShot()
{
	this->shot=rand()%2+1;
	if (this->shot==1)
	{
		this->score=true;	
	}
	else if(this->shot==2)
	{
		this->score=false;
	}
}
bool Player::getShot()
{
	return this->score;
	//if shot equals one make it true
	//if shot equals two make it false
	//if(this->shot==1){
	//	bool variable=True;
	//}
	//if(this->shot==2){
	//	bool variable=False;
	//}
}
void Player::getScore()
{
	//pass the values of shot to one and two 
	//and use that to determine what they get
	//if one is true and two is false
	//Player two gets an H unless he already has it then he gets a O and so on and so forth
	//if(bool variable==False)
	if(this->Horse[0]!='H')
	{
		this->Horse[0]='H';
	}
	else if(this->Horse[1]!='O')
	{
		this->Horse[1]='O';
	}
	else if(this->Horse[2]!='R')
	{
		this->Horse[2]='R';
	}
	else if (this->Horse[3]!='S')
	{
		this->Horse[3]='S';
	}
	else if (this->Horse[4]!='E')
	{
		this->Horse[4]='E';
	}
}		
char* Player::getHorse()
{
	return Horse;
}

File: Player.java
//Honor Pledge:
//
//I pledge that I have neither given nor
//received any help on this assignment.
//
//jascho
import java.io.*;
import java.util.Random;
import java.lang.String;
public class Player
{ 
	char print;
	//print given the value of the Horse array so I can print it in my main function
	char[] Horse;
	//holds the letters the players acquire
  	int shot;
	//gets a random number to determine whether the player made or missed the shot
	boolean score;
	//gets the value from shot so I can give it to a variable in my main
	public Player()
	{
		Horse = new char[5];
		//setting the character array Horse to something so that I can compare it when I run getScore
		Horse[0]='a';
		Horse[1]='b';
		Horse[2]='c';
		Horse[3]='d';
		Horse[4]='f';
		int shot=0;
		boolean score=false;
	}
	public void setShot()
	{
		//gives a random number to shot to determine whether or not they made the shot
		Random rand= new Random();
		shot=rand.nextInt(2)+1;
		if(shot==1)
		{
			score=true;
		}
		else if (shot==2)
		{
			score=false;
		}

	}
	public boolean getShot()
	{
		//returns whether or not they made the shot so I can give it to a variable in main to use in main
		return score;
	}
	public void getScore()
	{
		//gives the player a letter if they miss the shot
		if (Horse[0]!='H')
		{
			Horse[0]='H';
		}
		else if (Horse[1]!='O')
		{
			Horse[1]='O';
		}
		else if (Horse[2]!='R')
		{
			Horse[2]='R';
		}
		else if (Horse[3]!='S')
		{
			Horse[3]='S';
		}
		else if (Horse[4]!='E')
		{
			Horse[4]='E';
		}
	}
	public String getHorse()
	{
		//returns the letters the players has to main so I can print them
		String print= new String(Horse);
		return print;
	}
}
